azure-pipelines-official.yml

```diff
@@ -41,6 +41,26 @@ variables:
       value: $(dn-bot-devdiv-drop-rw-code-rw)
 
 stages:
+- stage: setup
+  displayName: Build Setup
+
+  jobs:
+  - job: SetBuildNumber
+    pool:
+      vmImage: vs2017-win2016
```

Comment:
having to dequeue a machine and clone the repo and all that stuff just to do this should help motivate us to get rid of this logic :)